utils/buildcli/README.md

```diff
@@ -0,0 +1,10 @@
+# builcli
```

Comment:
Yeah, I agree that it's a little cryptic. I was wondering what to name it, and consulted [PEP-0008](https://www.python.org/dev/peps/pep-0008/):

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

`buildcli` meets the "short, all-lowercase" criteria, but isn't very readable. How about `swift_build_support`?